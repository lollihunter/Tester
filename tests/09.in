20 20 10
1 1
14 2
5 7
10 5
13 8
18 11
1 3
11 7
17 3
20 20
