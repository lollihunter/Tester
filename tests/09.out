8
1 7 3 4 8 5 6 10 
